### PR TITLE
Simple perfomance boost

### PR DESCRIPTION
**Pull request by Hudell**
This change makes a great difference on performance in maps with many events.

### CODE REVIEW

**Comment by mateusfccp** [comment]
I think you should make a benchmark proving it, so we can consider.

**Comment by Hudell** [comment]
Sure.

File: src/rpg_sprites/Sprite_Character.js [hunk 1]
```diff
@@ -203,7 +203,13 @@ Sprite_Character.prototype.createHalfBodySprites = function() {
 Sprite_Character.prototype.updatePosition = function() {
     this.x = this._character.screenX();
     this.y = this._character.screenY();
-    this.z = this._character.screenZ();
+
+    var newZ = this._character.screenZ();
+    if (newZ !== this.z) {
```
Comment (quxios) [inline src/rpg_sprites/Sprite_Character.js:208]: Shouldn't you also request a sort if Y values also changed?

Comment (Hudell) [reply src/rpg_sprites/Sprite_Character.js:208]: I had first added a check on Game_CharacterBase.updateMove, but I guess keeping both conditions in the same class would make more sense.
I'll make a new commit changing it later.

File: src/rpg_objects/Game_Temp.js [hunk 2]
```diff
@@ -55,3 +56,16 @@ Game_Temp.prototype.destinationX = function() {
 Game_Temp.prototype.destinationY = function() {
     return this._destinationY;
 };
+
+Game_Temp.prototype.tileMapSortRequested = function() {
+  return this._tilemapSortRequested === true;
```
Comment (hajimehoshi) [inline src/rpg_objects/Game_Temp.js:61]: `return this._tilemapSortRequested` seems enough. On the other hand, there is no coding rule in this project yet, so I'd like to ask core maintainers of this project.

**Comment by ghost** [comment]
This makes performance boost when Character is not moving.

But in my experience, It's not good varying fps by condition.
Human tend to recognize peaky difference, not so on gradually difference.

I find out to change algorithm to 'Insertion Sort' makes great average performance boost on Android.
But It's not high priority to change, and It needs review.

**Pull request closed without merging**